src/dataExplorer/components/FluxQueryBuilder.tsx

```diff
@@ -1,38 +1,105 @@
 
 // Components
-import {DraggableResizer, Orientation} from '@influxdata/clockface'
+import {
+  AlignItems,
+  DraggableResizer,
+  FlexBox,
+  FlexDirection,
+  Orientation,
+} from '@influxdata/clockface'
 import {QueryProvider} from 'src/shared/contexts/query'
+import {EditorProvider} from 'src/shared/contexts/editor'
 import {ResultsProvider} from 'src/dataExplorer/components/ResultsContext'
 import ResultsPane from 'src/dataExplorer/components/ResultsPane'
+import SidePane from 'src/dataExplorer/components/SidePane'
 import Schema from 'src/dataExplorer/components/Schema'
 
 // Styles
 import './FluxQueryBuilder.scss'
+import LeftArrow from 'assets/images/collapse-left.svg'
+import RightArrow from 'assets/images/collapse-right.svg'
 
-const INITIAL_VERT_RESIZER_HANDLE = 0.2
+const INITIAL_LEFT_VERT_RESIZER_HANDLE = 0.2
+const INITIAL_RIGHT_VERT_RESIZER_HANDLE = 0.8
 
 const FluxQueryBuilder: FC = () => {
   const [vertDragPosition, setVertDragPosition] = useState([
-    INITIAL_VERT_RESIZER_HANDLE,
+    INITIAL_LEFT_VERT_RESIZER_HANDLE,
+    INITIAL_RIGHT_VERT_RESIZER_HANDLE,
   ])
+  const [showLeft, toggleLeftPanel] = useState(true)
+  const [showRight, toggleRightPanel] = useState(true)
+
+  const adjustMiddlePanel = grow => {
+    if (!grow) {
+      return
+    }
+    const el = document.querySelector(
+      '[data-testid="flux-query-builder-middle-panel"]'
+    )
+    if (el) {
+      (el as any).style.flexGrow = 0.8
+    }
+  }
 
   return (
     <QueryProvider>
-      <DraggableResizer
-        handleOrientation={Orientation.Vertical}
-        handlePositions={vertDragPosition}
-        onChangePositions={setVertDragPosition}
-      >
-        <DraggableResizer.Panel>
-          <Schema />
-        </DraggableResizer.Panel>
-        <DraggableResizer.Panel className="new-data-explorer-rightside">
-          <ResultsProvider>
-            <ResultsPane />
-          </ResultsProvider>
-        </DraggableResizer.Panel>
-      </DraggableResizer>
+      <EditorProvider>
+        <DraggableResizer
+          handleOrientation={Orientation.Vertical}
+          handlePositions={vertDragPosition}
+          onChangePositions={setVertDragPosition}
+        >
+          <DraggableResizer.Panel style={showLeft ? {} : {display: 'none'}}>
+            <Schema />
+          </DraggableResizer.Panel>
+          <DraggableResizer.Panel
+            testID="flux-query-builder-middle-panel"
+            className="new-data-explorer-rightside"
+          >
+            <FlexBox
+              direction={FlexDirection.Row}
+              alignItems={AlignItems.FlexStart}
+              style={{height: '100%', width: '100%', position: 'absolute'}}
+            >
+              <FlexBox.Child>
+                <span
+                  onClick={() => {
+                    toggleLeftPanel(!showLeft)
+                    adjustMiddlePanel(showLeft)
+                  }}
+                  style={{marginLeft: '-20px', float: 'left'}}
```

Comment:
Move the collapse icons to appear on top of the DraggableResizer's handle. Per design.

In the right side's divider, I had to set the zIndex to make it actionable (instead of the divider handle).